lms/djangoapps/email_marketing/tasks.py

```diff
@@ -17,6 +17,24 @@
 SAILTHRU_LIST_CACHE_KEY = "email.marketing.cache"
 
 
+@task(bind=True)
+def get_email_cookies_via_sailthru(self, post_parms):
+    try:
+        sailthru_client = SailthruClient(email_config.sailthru_key, email_config.sailthru_secret)
+        log.info(
+            'Sending to Sailthru the user interest cookie [%s] for user [%s]',
+            post_parms.get('cookies', ''),
+            user.email
+        )
+
+        sailthru_response = sailthru_client.api_post("user", post_parms)
+
+    except SailthruClientError:
```

Comment:
You should log the `SailthruClientError` here, then I think we can end the task with a `None` response?